CadRevealFbxProvider/Attributes/ScaffoldingAttributeParser.cs

```diff
@@ -326,6 +343,42 @@ string columnNameKeyAttribute
         return removeCsvRowsWithoutKeyAttribute;
     }
 
+    // returns true if there are items from multiple manufacturers in the attributes dictionary
+    // works only if "Manufacturer" column exists in the file, not for older templates
+    private static bool HasItemsFromMultipleManufacturers(
+        Dictionary<string, Dictionary<string, string>?> attributesDictionary
+    )
+    {
+        var manufacturers = attributesDictionary
+            .Where(a => a.Value != null)
+            .Select(av => av.Value!["Manufacturer"])
+            .Where(m => !string.IsNullOrEmpty(m))
+            .Distinct()
+            .ToList();
+        return manufacturers.Count > 1;
+    }
+
+    // returns true if there are items multiple manufacturers in the models
+    // this method should be used when the column Manufacturers is not present
+    // it looks at the description columns and checks if descriptions indicate several manufactuers
+    private static bool DetermineIfMultipleManufacturersFromDescription(
+        Dictionary<string, Dictionary<string, string>?> attributesDictionary
+    )
+    {
+        // prior to the template change, only Aluhak and HAKI were actually used in the models
+        bool hasHaki = attributesDictionary
+            .Where(a => a.Value != null)
+            .Select(av => av.Value!["Description"])
+            .Any(desc => desc.StartsWith("HAKI", StringComparison.OrdinalIgnoreCase));
+
+        bool hasOtherThanHaki = attributesDictionary
+            .Where(a => a.Value != null)
+            .Select(av => av.Value!["Description"])
+            .Any(desc => !desc.StartsWith("HAKI", StringComparison.OrdinalIgnoreCase));
+
+        return hasHaki && hasOtherThanHaki;
+    }
```

Comment:
I'd consider not supporting htese old models that are missing the manufacturer column.

Reply:
I realized now that we should definitely report old formats, and state that in this case, it can be a false negative!